content/rc/changelog/december-2023.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: Redis Cloud changelog (December 2023)
+linktitle: December 2023
+description: New features, enhancements, and other changes added to Redis Cloud during December 2023.
+highlights: Active-Active JSON support, Full mTLS support
+weight: 72
+alwaysopen: false
+categories: ["RC"]
+aliases: []
+---
+
+## New features
+
+### Active-Active JSON support
+
+[Active-Active databases]({{<relref "rc/databases/active-active-redis">}}) on Redis Cloud now support the [JSON]({{< relref "/stack/json" >}}) data type. 
+
+See [Create an Active-Active subscription]({{<relref "rc/subscriptions/create-active-active-subscription">}}) to learn how to create an Active-Active subscription.
+
+### Full mutual TLS (mTLS) support
```

Comment:
@cmilesb the changes include two improvements: 1) the customer can use multiple client certificates which should make it easier to rotate certificates once they expire 2) indeed specifying Trust CAs as you mention.

I think it would be better to change this title to "Enhancements for Mutual TLS support" or something like that. WDYT?